Automattic/wp-calypso
# Plans: Introduce a ProductSelector block

This introduces a `ProductSelector` block. The idea of this block is to be used on the plans page, and to replace the current static interface that lives there on the dev env for purchasing single plans.

It uses the newly introduced `ProductCard` component, and is going to use the new components for product options that @delawski is working on. It needs that component, which is why the PR is still in progress.

It also uses the existing products that have already been added to the WP.com store.

#### Changes proposed in this Pull Request

* Plans: Introduce a ProductSelector block

#### Preview

Desktop:
![](https://cldup.com/FBvSltsuZQ.png)

Mobile:
![](https://cldup.com/eTtQa8JmVi.png)

note: the screenshots are from before we renamed the component.

#### Testing instructions

* Checkout this branch
* Go to http://calypso.localhost:3000/devdocs/blocks/product-selector
* Test the tabs and verify the price changes.
* Verify the price comes from the products REST API request for the corresponding products.
* More instructions TBD soon.
